submissions/l1storez/popup/index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <title>Hooli-style Popup</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <nav class="top_bar">
```

Comment:
Please add a mobile version.
![image](https://user-images.githubusercontent.com/16196199/190487046-9b1fc120-f059-4644-bdad-e3aac98de681.png)

Reply:
Of course, I did not notice at once because on my phone everything is normal (412px wight)

Reply:
done!